frontend/src/v5/store/tickets/tickets.selectors.ts

```diff
@@ -122,3 +122,29 @@ export const selectStatusConfigByTemplateId = createSelector(
 	(state, ...args) => selectCurrentProjectTemplateById(state, args.at(-1)),
 	(ticketTemplate, projectTemplate) => ticketTemplate?.config?.status || projectTemplate?.config?.status || DEFAULT_STATUS_CONFIG,
 );
+
+export const selectAllValuesByModuleAndProperty = createSelector(
+	selectTemplates,
+	selectRiskCategories,
+	(state, modelId, module) => module,
+	(state, modelId, module, property) => property,
+	(state, modelId, module, property, type) => type,
+	(templates, riskCategories, module, property, type) => {
+		if (type === 'template') return templates.map(({ name }) => name);
+		if (!type) return;
```

Comment:
Im inclined to put this in a different selector because type is not used elsewhere in the selector its kind of used as a boolean only (type=='template)